### PR TITLE
Exp 27 09 branch

### PR DESCRIPTION
**Pull request by YanisLalou**
For DANN:
+ After some exp, seems like the network is not learning when we have lr_classifier = 10* lr_feature_layer. So we revert it. 
+ Also usually the lr should be 1e-2 but with the lr_scheduler keeping it at 1 for office home seems to showcase similar results.

### CODE REVIEW

File: solvers/dann.py [hunk 1]
```diff
@@ -57,8 +57,8 @@ def get_estimator(self, n_classes, device, dataset_name, **kwargs):
             domain_classifier=DomainClassifier(num_features=params['num_features']),
             optimizer__param_groups=[
                 ('base_module_.feature_layer*', {'lr': params['lr']}),
-                ('base_module_.final_layer*', {'lr': 10*params['lr']}),
-                ('domain_classifier_*', {'lr': 10*params['lr'],}),
+                ('base_module_.final_layer*', {'lr': params['lr']}),
+                ('domain_classifier_*', {'lr': params['lr'],}),
```
Comment (antoinecollas) [inline solvers/dann.py:61]: Why do you need to change this?

Comment (antoinecollas) [reply solvers/dann.py:61]: Sorry, it's in the PR ^^

File: datasets/officehome.py [hunk 2]
```diff
@@ -20,17 +20,17 @@ class Dataset(BaseDataset):
     parameters = {
         'source_target': [
             ('art', 'clipart'),
-            ('art', 'product'),
```
Comment (tgnassou) [inline datasets/officehome.py:23]: Re-add everything for the next run

File: solvers/deep_coral.py [hunk 3]
```diff
@@ -24,7 +25,7 @@ class Solver(DeepDASolver):
     # the cross product for each key in the dictionary.
     # All parameters 'p' defined here are available as 'self.p'.
     default_param_grid = {
-        'criterion__reg': [1, 0.1, 0.01],
+        'criterion__reg': np.logspace(-5, 3, 9),
```
Comment (tgnassou) [inline solvers/deep_coral.py:28]: 10e3 seems big for me but why not

**Comment by tgnassou** [comment]
LGTM

**Pull request merged**